cmake/use-fetch-content.cmake

```diff
@@ -0,0 +1,175 @@
+cmake_minimum_required(VERSION 3.24)
+
+if(NOT BEMAN_EXEMPLAR_LOCKFILE)
+    set(BEMAN_EXEMPLAR_LOCKFILE
```

Comment:
Why not use options here? Is it because CACHE FILE PATH?

Reply:
option is only intended for boolean configs

Reply:
thanks for the context!